### PR TITLE
Update price

### PR DESCRIPTION
**Pull request by techeng322**

### CODE REVIEW

File: hooks/useTBAPurchase.tsx [hunk 1]
```diff
@@ -16,6 +16,18 @@ const useTBAPurchase = () => {
   const [loading, setLoading] = useState(false)
   const { isLoggedByEmail } = useUserProvider()
   const { prepare } = usePreparePrivyWallet()
+  const [preparedMulticalls, setPreparedMulticalls] = useState(null)
+
+  useEffect(() => {
+    const init = async () => {
+      const prepared = await getPreparedMulticalls(
+        (connectedWallet as `0x${string}`) || "0x0000000000000000000000000000000000000000",
```
Comment (sweetmantech) [inline hooks/useTBAPurchase.tsx]: can you update to use `zeroAddress` from `viem`?
- replace hard-coded magic address 🪄 
```suggestion
        (connectedWallet as `0x${string}`) || zeroAddress,
```

File: hooks/useTBAPurchase.tsx [hunk 2]
```diff
@@ -16,6 +17,18 @@ const useTBAPurchase = () => {
   const [loading, setLoading] = useState(false)
   const { isLoggedByEmail } = useUserProvider()
   const { prepare } = usePreparePrivyWallet()
+  const [preparedMulticalls, setPreparedMulticalls] = useState(null)
+
+  useEffect(() => {
+    const init = async () => {
+      const prepared = await getPreparedMulticalls(
+        (connectedWallet as `0x${string}`) || zeroAddress,
```
Comment (sweetmantech) [inline hooks/useTBAPurchase.tsx]: can you use `Address` type from viem as well?
```suggestion
        (connectedWallet as Address) || zeroAddress,
```

Comment (techeng322) [reply hooks/useTBAPurchase.tsx]: fixed ✅

File: hooks/useTBAPurchase.tsx (hunk 1 above)
Comment (techeng322) [reply hooks/useTBAPurchase.tsx]: fixed ✅

**Review by sweetmantech — APPROVED** [review]

**Pull request merged**